Formula/d/docker-buildx.rb

```diff
@@ -32,11 +33,18 @@ def install
     generate_completions_from_executable(bin/"docker-buildx", "completion")
   end
 
+  def post_install
+    cli_plugins = HOMEBREW_PREFIX/"lib/docker/cli-plugins"
```

Comment:
Could you motivate this choice of path? Is `lib/docker/cli-plugins` a "well-known" path (in some sense)?

Reply:
Yes, following the original design https://github.com/docker/cli/issues/1534 docker-cli scans these paths, when it searches for plugins:

> Unix-like OSes:
> - `$HOME/.docker/cli-plugins`
> - `/usr/local/lib/docker/cli-plugins` & `/usr/local/libexec/docker/cli-plugins`
> - `/usr/lib/docker/cli-plugins` & `/usr/libexec/docker/cli-plugins`

I think it's reasonable to keep Docker CLI's plugins, that brew manages, in the similar structure.

Note, Docker Desktop seems to amend the default list of the paths and bundles several cli-plugins into the distribution itself (e.g. buildx and compose live inside the docker desktop's bundle).